requirements.txt

```diff
@@ -1,5 +1,6 @@
-Django==3.0.7
+Django==3.2.7
```

Comment:
Changing these may have side effects. Try not to do so unless required.

Reply:
Certain python versions aren't supported in M1, hence had to change Django version to suit compatibility.